lib/membrane/aac/parser.ex

```diff
@@ -55,8 +72,28 @@ defmodule Membrane.AAC.Parser do
     end
   end
 
+  def handle_process(:input, buffer, ctx, state) when state.in_encapsulation == :none do
+    # Since there is no ADTS header, there is nothing to parse on the input
+    # Therefore, we only really add the timestamp
+    timestamp = Helper.next_timestamp(state.timestamp, ctx.pads.input.caps)
+    metadata = Map.put(buffer.metadata, :timestamp, timestamp)
+
+    buffer =
+      %{buffer | metadata: metadata}
+      |> then(&ensure_out_encapsulation(&1, ctx.pads.input.caps, state))
```

Comment:
Maybe 
```elixir
buffer = %{buffer | metadata: metadata}

buffer =
if state.out_encapsulation == :ADTS do
  %Buffer{buffer | payload: Helper.to_adts(caps, buffer.payload)}
else 
 buffer
end
``` 

I think it is easier to read than moving to a separate function.
Also ensure_out_encapsulation doesn't need to check if `in_encapsulation` is set to `:none` as we already checked this in function definition